certificate-manager/src/test/java/io/strimzi/certs/SecretCertProviderTest.java

```diff
@@ -74,7 +74,7 @@ public void testAddKeyAndCertInSecret() throws IOException {
 
         ssl.generateSelfSignedCert(key, cert, 365);
 
-        Secret secret = secretCertProvider.createSecret("my-namespace", "my-secret", key, cert, Collections.emptyMap(), ownerReference);
+        Secret secret = secretCertProvider.createSecret("my-namespace", "my-secret", key, cert, Collections.emptyMap(), Collections.emptyMap(), ownerReference);
```

Comment:
Long line, static import

Reply:
I have used the static import and made the line shorter but tbh I don't like them :-)
Static import makes the code less readable because you don't know what is the class which holds the method you are calling. Taking a look at the code `emptyMap` could be a method from the current class while it's part of `Collections` instead. But it's just my view on static imports.

Reply:
I understand that PoV, but I think that's just an argument about being thoughtful about which methods you import statically. For sure I wouldn't advocate using static imports for all static methods. For things like `emptyList()` and `singletonList()`, which are _really common_, I think it's pretty obvious what method is being called. Likewise in tests, it's just much nicer to read `assertTrue()` because it's less noisy.

Reply:
In this terms I agree with you then.